.devcontainer/dodona-tested.dockerfile

```diff
@@ -65,6 +65,15 @@ RUN <<EOF
         xxd \
         shellcheck
 
+    # JavaScript dependencies
+    bash -c 'set -o pipefail && curl -fsSL https://deb.nodesource.com/setup_22.x | bash -'
+    apt-get install -y --no-install-recommends nodejs
+    npm install -g eslint@8.57 abstract-syntax-tree@2.22
+
+    # TypeScript dependencies
+    npm install -g typescript@5.6.3 tsx@4.19.2
+    npm install -g @types/node @typescript-eslint/parser @typescript-eslint/eslint-plugin --save-dev
```

Comment:
I don't think `--save-dev` should be added here. Eslint is used in production by the linter function you wrote.

Reply:
This also was something I forgot to check if it should be removed or not. I removed it.